moped-database/views/uniform_features.sql

```diff
@@ -1,62 +1,98 @@
--- latest version 1678894870998_remove_cols_from_geo_views
+-- latest version from 1691698449987_add-district-columns-to-views
 DROP VIEW "public"."uniform_features";
 
-CREATE OR REPLACE VIEW "public"."uniform_features" AS
+CREATE VIEW "public"."uniform_features" AS
 SELECT
     feature_signals.id,
     feature_signals.component_id,
     'feature_signals'::text AS "table",
-    json_build_object(
-        'signal_id', feature_signals.signal_id, 'knack_id', feature_signals.knack_id, 'location_name', feature_signals.location_name, 'signal_type', feature_signals.signal_type) AS attributes,
-    feature_signals.geography
+    json_build_object('signal_id', feature_signals.signal_id, 'knack_id', feature_signals.knack_id, 'location_name', feature_signals.location_name, 'signal_type', feature_signals.signal_type) AS attributes,
+    feature_signals.geography,
+    districts.council_districts
```

Comment:
it seems to make sense to leave `council_districts` outside of the `attributes` object, since that object contains properties that are specific to this feature type.  this comment will hopefully make more sense if you inspect the rows returned from the `uniform_features` or `project_geography` 😅

Reply:
agreed! doesn't feel like a good fit in attributes 👍